components/server/src/main.ts

```diff
@@ -13,8 +13,8 @@ import { productionContainerModule } from "./container-module";
 import { dbContainerModule } from "@gitpod/gitpod-db/lib/container-module";
 
 const container = new Container();
-container.load(productionContainerModule);
 container.load(dbContainerModule);
+container.load(productionContainerModule);
```

Comment:
This should be ok as it makes no differenece.
Still I wonder whether why it was necessary: if it really was, that feels like a potential bigger problem.

Reply:
Ah, thanks for explaining! :pray: ([relevant line](https://github.com/gitpod-io/gitpod/pull/17906/files#diff-e354e39f89620df3c7e7c02f79ea6938e819c08bf57e6ade31160f3dad9ce221R55))